src/components/RuntimeManager.js

```diff
@@ -226,7 +226,7 @@ export default class RuntimeManager extends PureComponent {
     } else if (runtime.status === 'Running' && prevRuntime.status === 'Updating') {
       notify('success', 'Number of workers has updated successfully.')
     }
-    if (prevApp && prevApp.status !== 'RUNNING' && app && app.status === 'RUNNING') {
+    if (prevApp?.status !== 'RUNNING' && app?.status === 'RUNNING') {
```

Comment:
Something is not working with this logic. When switching between tabs if Galaxy is running a new "launch galaxy" notification pops up on every tab switch which it was not previously doing. (@Shakespeared )
<img width="1530" alt="Screen Shot 2021-05-10 at 3 13 09 PM" src="https://user-images.githubusercontent.com/54322292/117712466-6018fa00-b1a2-11eb-81d4-54640c6b0383.png">

Reply:
Good catch, I re-ordered it to be clearer.